src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -95,6 +100,51 @@ public boolean isEnabled(final String toggleName, final UnleashContext context ,
         return enabled;
     }
 
+    @Override
+    public Variant getVariant(String toggleName, UnleashContext context) {
+        return getVariant(toggleName, context, "");
+    }
+
+    @Override
+    public Variant getVariant(String toggleName, UnleashContext context, String defaultPayload) {
```

Comment:
The user should be allowed to inject a default `Variant`, not just the payload.

Reply:
Fixed.